code/dqe/rdbhdbcompare.q

```diff
@@ -0,0 +1,9 @@
+\d .dqe
+rdbhdbcompare:{[tab]
+  hdbmeta:(first exec w from .servers.SERVERS where procname=`hdb1)(meta;tab);
```

Comment:
Rather than directly querying `.servers.SERVERS`, I think you should use the `.servers.getservers` function